lib/dynamodb-integration.js

```diff
@@ -10,12 +9,12 @@ const BASE_EVENT_SIZE_BYTES = 26;
 
 const debug = require('./utils').debug;
 
-const lib = {};
+const lib = { MAX_BATCH_ITEM_NUM: 25 };
```

Comment:
Can you please explain why you did this?

Reply:
For the export. We currently export lib as is. I did that instead of splatting